enterprise/core-edge/src/main/java/org/neo4j/coreedge/core/CoreEdgeClusterSettings.java

```diff
@@ -190,71 +190,101 @@
     public static final Setting<Long> edge_time_to_live =
             setting( "core_edge.edge_time_to_live", DURATION, "1m", min(60_000L) );
 
-    @Description( "Edge server 'call home' frequency"  )
+    @Description( "How often edge servers should contact core servers for refreshing database contents"  )
     public static final Setting<Long> edge_refresh_rate =
             setting( "core_edge.edge_refresh_rate", DURATION, "5s", min(5_000L) );
 
     @Description( "How long drivers should cache the data from the `dbms.cluster.routing.getServers()` procedure."  )
     public static final Setting<Long> cluster_routing_ttl =
             setting( "core_edge.cluster_routing_ttl", DURATION, "5m", min(1_000L) );
 
-    @Description( "NODE ID Allocation Space Size" )
+    @Description( "The size of the id allocation requests core servers will make when they runs out of NODE ids. " +
+            "Larger values mean less frequent requests but also result in more unused ids (and unused disk space) " +
+            "in the event of a crash." )
     public static final Setting<Integer> node_id_allocation_size =
             setting( "core_edge.node_id_allocation_size", INTEGER, "1024" );
 
-    @Description( "RELATIONSHIP ID Allocation Space Size" )
+    @Description( "The size of the id allocation requests core servers will make when they run out " +
+            "of RELATIONSHIP ids. Larger values mean less frequent requests but also result in more unused ids " +
+            "(and unused disk space) in the event of a crash." )
     public static final Setting<Integer> relationship_id_allocation_size =
             setting( "core_edge.relationship_id_allocation_size", INTEGER, "1024" );
 
-    @Description( "PROPERTY ID Allocation Space Size" )
+    @Description( "The size of the id allocation requests core servers will make when they run out " +
+            "of PROPERTY ids. Larger values mean less frequent requests but also result in more unused ids " +
+            "(and unused disk space) in the event of a crash." )
     public static final Setting<Integer> property_id_allocation_size =
             setting( "core_edge.property_id_allocation_size", INTEGER, "1024" );
 
-    @Description( "STRING_BLOCK ID Allocation Space Size" )
+    @Description( "The size of the id allocation requests core servers will make when they run out " +
+            "of STRING_BLOCK ids. Larger values mean less frequent requests but also result in more unused ids " +
```

Comment:
Re "STRING_BLOCK": beginning to suspect that "NODE", "RELATIONSHIP" etc have a particular, perhaps different meaning in this context. If so the all-caps may be appropriate. We should then add a section elsewhere explaining what these are and how to reason about these settings.